### PR TITLE
feat:  142 update password auth workflow

### LINKED ISSUE #142: Update password auth workflow
Change password as an authenticated user

### PR DESCRIPTION
**Pull request by qcdyx**
**Summary:**
Closes #142 
Finish the UI of Change password screen

**Expected behavior:** 
image

**Testing tips:**

Provide tips, procedures and sample files on how to test the feature.
Testers are invited to follow the tips AND to try anything they deem relevant outside the bounds of the testing tips. 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `./scripts/api-tests.sh` to make sure you didn't break anything
- [x] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- [x] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

File: web-app/src/app/router/Router.tsx [hunk 1]
```diff
@@ -23,6 +24,7 @@ export const AppRouter: React.FC = () => {
         <Route path='account' element={<Account />} />
       </Route>
       <Route path='contact-info' element={<ContactInformation />} />
+      <Route path='change-password' element={<ChangePassword />} />
```
Comment (davidgamez) [inline web-app/src/app/router/Router.tsx:27]: This should be a protected screen.
```suggestion
<Route element={<ProtectedRoute />}>
      <Route path='change-password' element={<ChangePassword />} />
</Route>
```

**Review by emmambd — CHANGES_REQUESTED** [review]
2 pieces of feedback on the UI: 

- Change password form should sit in the middle of the page like https://qa.mobilitydatabase.org/sign-up (it might be already, but I can't tell since I can't see the nav items in the screenshot) 
- There should be a toggle view icon option for the New Password and Confirm New Password fields so people can see their new password as they're writing it.

**Review by emmambd — APPROVED** [review]
LGTM from standup!

**Comment by qcdyx** [comment]
I'll remove my credentials once test is passed.

**Comment by qcdyx** [comment]
I'm researching on the firebase auth emulator so that I can test Firebase SDK end-to-end workflows.

**Review by davidgamez — APPROVED** [review]
LGTM!

File: web-app/src/app/store/profile-reducer.ts [hunk 2]
```diff
@@ -80,6 +85,7 @@ export const userProfileSlice = createSlice({
     },
     logoutSuccess: (state) => {
       state.status = 'unauthenticated';
+      state.isSignedInWithProvider = false;
```
Comment (davidgamez) [inline web-app/src/app/store/profile-reducer.ts:88]: 👍

**Pull request merged**